java/test/jmri/PathLengthTest.java

```diff
@@ -88,9 +88,17 @@ public static Test suite() {
         return suite;
     }
 
-    @Override
+    @Before
```

Comment:
If you are adding the annotations @Before and @After, the annotated method must be public.

Also, the junit.framework.* imports should be removed, sinvpce this is no longer in junit 3 syntax.

Reply:
Thanks for checking. That one's a failure of my migration scripts; didn't properly catch that it was JUnit3 still.